src/promptflow/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - [SDK/CLI][azure] Create a run with `resume_from`.
   - CLI: Support `pfazure run create --resume-from <original-run-name>` to create a run resume from another run.
   - SDK: Support `pf.run(resume_from=<original-run-name>)` to create a run resume from another run.
+- [SDK] Support flow exectue as async function: `load_flow(<flow>, is_async_call=True)` will return an async callable flow object.
```

Comment:
change the docstring of load_flow function?

Reply:
let me check.

Reply:
Good catch.